torch_xla/csrc/ops/dynamic_ir.h

```diff
@@ -38,40 +38,50 @@ namespace torch_xla {
 class SizeNode : public XlaNode, public torch::lazy::DimensionNode {
```

Comment:
Irrelevant to this pr, but at this line 27 in this file, should it be `numel = x.size()[0] * x.size()[1]`? I couldn't find a `size()` for a tensor.

Reply:
yea you might be right, fell free to send a pr to fix the comment